include/vga.h

```diff
@@ -310,7 +316,14 @@ struct VGA_TANDY {
 struct VGA_Seq {
 	uint8_t index = 0;
 	uint8_t reset = 0;
-	uint8_t clocking_mode = 0;
+
+	union {
+		uint8_t data = 0;
+		// Characters are drawn 8 pixels wide (or 9 if cleared)
+		bit_view<0, 1> is_eight_dot_mode;
+		bit_view<3, 1> is_pixel_doubling;
+	} clocking_mode = {};
```

Comment:
Are we storing the contents of a 8-bit-long VGA register here or something? The very particular bit-layout implies that, but then it's chucked in the middle of `VGA_Seq` which seems (?) to be just some struct we made up in the emulation side... or maybe not, and the whole `VGA_Seq` struct maps to a specific VGA register/memory layout? I'm confused.

I know it's existing code, but maybe some clarification would be good in the form of comments if you managed to understand this a bit better 😄

Reply:
Will add docs :+1: 

It's an 8-bit register set by CRTC side and used on the VGA drawing side to indicate if (9/8 dot-mode) and pixel-doubling. 

The `bit_view` eliminates the magic bit-masking literals.

Also see why it looks strange amoung other bare `uint8_t`; the reality is, all of the other registers could be switched over to bit_view's too.

The reason I only "upgraded" this one is because I had to touch the code that checks these bits.

Reply:
Oh yeah, that's a good improvement, and I'm not asking it to bitview everything 😅 A simple explanation would be enough saying that these uint8 values are actually storing VGA register data. Could be useful to cluesless folks like myself 😄